src/extras/DataUtils.js

```diff
@@ -164,13 +164,106 @@ function fromHalfFloat( val ) {
 
 }
 
+// RGB9E5 packing/unpacking
+
+const RGB9E5_MANTISSA_BITS = 9;
+const RGB9E5_EXP_BIAS = 15;
+const RGB9E5_MAX_VALID_BIASED_EXP = 31;
+const MAX_RGB9E5 = ( RGB9E5_MAX_VALID_BIASED_EXP - RGB9E5_EXP_BIAS );
```

Comment:
This expression defines `MAX_RGB9E5_EXP`. The `MAX_RGB9E5` value uses a different expression and should evaluate to 65408.

Reply:
Thanks! Should be fixed via 59fb414eceb54d8e92816967a1d9dd62fd3997e3.